roles/image_build_infra/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+image_build_terraform_project_path: "{{ playbook_dir }}/terraform-caas-image-build"
+image_build_cluster_name: "caas-image-build"
+
+# Regex to capture existing cloud image names to use as the
+# OpenHPC Slurm base-image
+image_build_existing_image_regex: "^Rocky-8-GenericCloud-Base-8.8-.*"
```

Comment:
I'm not convinced that a control node built with an image based off anything after an 8.6 GenericCloud image will survive a reboot - the device labling appears to get screwed up for VMs with attached volumes.

See https://forums.rockylinux.org/t/issue-with-8-7-genericcloud-image-with-attached-volumes/8451 and https://stackhpc.slack.com/archives/C01LHHGF6BC/p1672999238711989.

Reply:
Tested on a running cluster with no issues, thanks for the heads up though